### ISSUE TITLE
msPointerEnabled should be used for IE

### ISSUE THREAD

**Issue by johnd0e** [Leaflet/Leaflet#7028]
https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/dev-guides/hh673557(v=vs.85)?redirectedfrom=MSDN#feature-detection

See https://github.com/openseadragon/openseadragon/issues/571

**Comment by johnd0e** [comment]
Ok, as I see in past we had exact such code, but that was changed by afa0d1d87b4c522ca3cbeef785c829b886355982 (c) @IvanSanchez

**Comment by jonkoops** [comment]
I don't think we want to fix this issue as we are dropping support for (older) versions of Internet Explorer in the future. Adding changes to support this unused browser makes no sense at this point.
